libsnowflakeclient/deps-build/darwin/curl/lib/pkgconfig/libcurl.pc

```diff
@@ -23,7 +23,7 @@
 # This should most probably benefit from getting a "Requires:" field added
 # dynamically by configure.
 #
-prefix=/bamboo/snowflake/snowflake-odbc/Installer/../Dependencies/linux/curl
+prefix=/Users/bamboo/Desktop/harry/libsnowflakeclient/scripts/../deps-build/darwin/Release/curl
```

Comment:
Isn't this Referring to a local Desktop location. This should be relative to the base directory.